src/admin/views/NewBannerView.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {Helmet} from 'react-helmet';
+import {useMutation} from '@apollo/react-hooks';
```

Comment:
You want to update this to import from @apollo/client - import {useMutation} from '@apollo/client';